pkg/credentials/aws.go

```diff
@@ -0,0 +1,33 @@
+package credentials
+
+import "fmt"
+
+const (
+	// AwsCredentialsStringFormat format strings for printing AWS credentials as a string or as environment variables
+	AwsCredentialsStringFormat = `Temporary Credentials:
```

Comment:
Really Minor Nitpick: Maybe use `\t` to indent the lines in the String. Just from looking at it in the code it's a bit confusing right now as it's the same indentation level as the surrounding code.

Reply:
This was all copy-paste from what used to be in the old `console` and `credentials` commands, so I wanted to keep that identical.

I also don't believe these should be indented. Right now this will all print to a terminal as such:
```
Temporary Credentials:
AccessKeyID: test-key
SecretAccessKey: test-secret
SessionToken: test-token
Region: test-region
Expires: test-expiration
```

I believe that's what we'd want.

Reply:
It seems we don't have a good way to make indentation look nice in raw string.  https://github.com/golang/go/issues/32590

Let's keep it as is, as long as the output is what we want.

Reply:
Non-blocking - I wonder if we could use go's text/templates here to better define these strings separately.